src/client/client4.ts

```diff
@@ -749,7 +749,7 @@ export default class Client4 {
         );
     };
 
-    getProfilesInChannel = (channelId: string, page = 0, perPage = PER_PAGE_DEFAULT, sort = '') => {
+    getProfilesInChannel = (channelId: string, page = 0, perPage = PER_PAGE_DEFAULT, sort = '', options = {}) => {
```

Comment:
Can we maybe get a type definition for what `options` can be?

Reply:
I followed the similar approaches in the similar APIs. It can be basically anything with a key-value pair.

Reply:
Added.